client/app/lib/components/DataTable.jsx

```diff
@@ -0,0 +1,89 @@
+import { createTheme, ThemeProvider } from '@mui/material';
```

Comment:
I assume this will be rebased once my branch is merged?

Reply:
Yes. I think I might have added some minor changes but in general I referenced this from your branch.